Week2/num1.js

```diff
@@ -0,0 +1,13 @@
+
+let greeting = "Hellow World" // greeting in many languages
```

Comment:
Overall the code looks good! There are just some small things here and there!

Keep it up!